.drone.star

```diff
@@ -199,9 +199,9 @@ def main(ctx):
     ]
 
   elif \
-  (ctx.build.event == "pull" and '[docs-only]' in ctx.build.title) \
+  (ctx.build.event == "pull_request" and '[docs-only]' in ctx.build.title) \
```

Comment:
why that combination? IMO the reduced pipeline for docs only should also run on master

Reply:
ah I see, comes from here https://github.com/owncloud/ocis/commit/e2817731fcad52091d686afb424aebc3a5cfee08

I think we need to clarify what should happen.

Reply:
It is run on master too. The problem is, that ctx.build.title is the commit message for commits and the PR descriptions for PRs.
You would not expect a docs-only pipeline if you just write something about [docs-only] in your PR description. This happend to me and did cost me half a day to find out what happened.

Reply:
I thought `# [docs-only] is not taken from PR messages, but from commit messages` does explain it

Reply:
be careful, drone takes everything from the PR description. for example [skip-ci]